internal/cmd/shim.go

```diff
@@ -69,6 +71,42 @@ func pluginPythonCode(s config.SQLPython) *plugin.PythonCode {
 	}
 }
 
+func pluginGoCode(s config.SQLGo) *plugin.GoCode {
+	return &plugin.GoCode{
+		EmitInterface:             s.EmitInterface,
+		EmitJsonTags:              s.EmitJSONTags,
+		EmitDbTags:                s.EmitDBTags,
+		EmitPreparedQueries:       s.EmitPreparedQueries,
+		EmitExactTableNames:       s.EmitExactTableNames,
+		EmitEmptySlices:           s.EmitEmptySlices,
+		EmitExportedQueries:       s.EmitExportedQueries,
+		EmitResultStructPointers:  s.EmitResultStructPointers,
+		EmitParamsStructPointers:  s.EmitParamsStructPointers,
+		EmitMethodsWithDbArgument: s.EmitMethodsWithDBArgument,
+		JsonTagsCaseStyle:         s.JSONTagsCaseStyle,
+		Package:                   s.Package,
+		Out:                       s.Out,
+		SqlPackage:                s.SQLPackage,
+		OutputDbFileName:          s.OutputDBFileName,
+		OutputModelsFileName:      s.OutputModelsFileName,
+		OutputQuerierFileName:     s.OutputQuerierFileName,
+		OutputFilesSuffix:         s.OutputFilesSuffix,
+	}
+}
+
+func pluginGoType(o config.Override) *plugin.ParsedGoType {
+	// Note that there is a slight mismatch between this and the
+	// proto api. The GoType on the override is the unparsed type,
+	// which could be a qualified path or an object, as per
+	// https://docs.sqlc.dev/en/latest/reference/config.html#renaming-struct-fields
```

Comment:
kind of yucky but works right

Reply:
note that this causes later:
```diff
-	for _, oride := range settings.Overrides {
- 		if oride.GoTypeName == "" {
+ 	for _, oride := range req.Settings.Overrides {
+ 		if oride.GoType.TypeName == "" {
```

which i don't love - welcome naming suggestion

Reply:
that was my read as well - yes, TypeName should be the parsed type name, not the full path